### ISSUE TITLE
Albert-base-v2 training data

### ISSUE THREAD

**Issue by slvcsl** [google-research/albert#225]
Which training data is used for albert in v2? 
You mention that "In this version, we apply 'no dropout', 'additional training data' and 'long training time' strategies to all models". Thanks
